### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by chrisknight32** [agronholm/anyio#726] [General]
Hello!

What is the expected release schedule? There are several bug fixes in main that I'm interested in that don't seem to be in a release yet. It's also been several months since a release so was wondering what the expected cadence is or what the eta is for the next release?

Thanks for your thoughts/work!

**Comment by agronholm** [comment]
I need at least #717, #714 and a fix for #695 (which I'm working on) to cut the next release.

**Reply by agronholm** [reply]
#735 (which is currently being reviewed) and #695 are the only release blockers left.

**Reply by agronholm** [reply]
v4.4.0 is out.

**Reply by chrisknight32** [reply]
Thank you so much for the fixes and the release!
